cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/dichotomy/MultipleDichotomyRunner.java

```diff
@@ -99,6 +128,23 @@ public MultipleDichotomyResult<DichotomyRaoResponse> runMultipleDichotomy(CseReq
                         dichotomyResultHelper.computeLowestUnsecureItalianImport(multipleDichotomyResult.getBestDichotomyResult());
                     double newLowestUnsecureItalianImport = dichotomyResultHelper.computeLowestUnsecureItalianImport(nextDichotomyResult);
 
+                    if (flattenPrasIds(forcedPrasIds) != null) {
+                        printablePrasIdsIds = toString(flattenPrasIds(forcedPrasIds));
+                    }
+                    String printableAdditionalPrasToBeForced = "";
+                    printableAdditionalPrasToBeForced = toString(additionalPrasToBeForced);
+                    businessLogger.info(SUMMARY,
+                        dichotomyCount,
+                        round(dichotomyResultHelper.computeHighestSecureItalianImport(nextDichotomyResult)),
+                        TtcResult.limitingCauseToString(nextDichotomyResult.getLimitingCause()),
+                        newLimitingElement,
+                        printablePrasIdsIds,
+                        printableAdditionalPrasToBeForced);
```

Comment:
Again here, the first one you should retrieve them from the dichotomy result and the second one should be printablePrasIdsIds